src/node.js

```diff
@@ -14,6 +14,16 @@ define([
     return blockElementNames.includes(node.nodeName);
   }
 
+  // Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elemente
+  var inlineElementNames = Immutable.Set.of('B', 'BIG', 'I', 'SMALL', 'TT',
```

Comment:
Yes, sure! I assume there will be others with time (like `blockElementNames` for instance), what do you think we put them in a folder, say `/src/vars` or `/src/node/vars`?